horovod/ray/elastic_v2.py

```diff
@@ -177,6 +179,7 @@ class ElasticParams(BaseParams):
     min_workers: int = 1
     max_workers: int = None
     reset_limit: int = None
+    cooldown_range: List[int] = None
```

Comment:
This implies that any size is allowed, where we only allow for exactly two ints. Can this be changed to `Tuple[int, int]`, or even `Optional[Tuple[int, int]]`?

Reply:
Done